modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -191,5 +191,12 @@ def veteran_file_number(auto_claim)
     def evss_service
       ClaimsApi::EVSSService::Base.new
     end
+
+    def rescue_generic_errors(power_of_attorney, e)
+      power_of_attorney.status = ClaimsApi::PowerOfAttorney::ERRORED
+      power_of_attorney.vbms_error_message = e&.message || e&.original_body
+      power_of_attorney.save
+      ClaimsApi::Logger.log('PoaVBMSUploadJob', message: "In generic rescue, the error is: #{e}")
```

Comment:
Here and in https://github.com/department-of-veterans-affairs/vets-api/pull/18397: should we change `'PoaVBMSUploadJob'` to be something generic?

Reply:
@tycol7 Good call! Do you mind re-stamping?